### ISSUE TITLE
Create a single non cluttered solution for all the trigger buttons next to user names.

### ISSUE THREAD

**Issue by creesch** [toolbox-team/reddit-moderator-toolbox-redesign#91]
This is gets cluttered fast, even more so if we start adding [metrics](#650) as well. 
![](https://i.imgur.com/F2ieEv9.png)

This would be sort of an solution but isn't ideal since it might be entirely unclear for the user that there is a dropdown under notes. 
![](http://i.imgur.com/C05MRcz.png)

This issue needs to be resolved before we tackle #650

**Comment by creesch** [comment]
Transfered this though in redesign we already have a solution in the form of the user hover popup. On old reddit there is the RES and reddit native popup on hover, neither can be accessed easily so that remains an issue.

**Comment by Geo1088** [comment]
Can we just spoof the `userHovercard` event in native popups on old Reddit and call this done, or do we still want to make our own dropdown-style thing for this?

**Comment by creesch** [comment]
Last time I checked it was a bit difficult to detect the popup on old reddit and more importantly link it to the the post that caused to trigger it. Then there is RES that also has a popup (and I think the ability to hide the reddit native one) which might cause some confusion. 

So I am not sure if there is a good clean solution to using those. In that regard I am still more inclined to implement something ourselves. Or, leave it as is and close this issue to be frank.

**Comment by Geo1088** [comment]
> and more importantly link it to the the post that caused to trigger it.

Right, I hadn't considered this... The popup doesn't get inserted near the post in DOM does it? So we'd have to figure out which element the user is hovering and figure it out that way somehow... Ugh.

I'm also kinda inclined to just close this issue and clarify that the existing "Show toolbox buttons in hovercard" setting only applies to new Reddit.
